draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1262,6 +1238,12 @@ the media type is encoded as an SD-JWT.
   </front>
 </reference>
 
+<reference anchor="EUDIW.ARF" target="https://digital-strategy.ec.europa.eu/en/library/european-digital-identity-wallet-architecture-and-reference-framework">
```

Comment:
Even if It May sounds paradoxal, this links Is not updated, I'll write today asking to update the arf version, since It Is still v1.0.0 while v1.2.0 Is going to be published

FYI below the fresh meat
https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/release

Reply:
nice! thank you!